### PR TITLE
Feature/Add Rspec Tests

### PR DESCRIPTION
**Pull request by amiranda2510**
add to gemfile and configure rspec and simplecov

### CODE REVIEW

**Comment by amiranda2510** [comment]
image

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.tabSize": 2
```
Comment (DeadKennedyx) [inline .vscode/settings.json:2]: add this file to the git ignore

**Review by DeadKennedyx — CHANGES_REQUESTED** [review]
add the coverage folder to git ignore

File: .vscode/settings.json (hunk 1 above)
Comment (amiranda2510) [reply .vscode/settings.json:2]: solved

**Pull request merged**